gridstatus/pjm.py

```diff
@@ -532,6 +624,450 @@ def _get_key(self):
 
         return settings["subscriptionKey"]
 
+    @staticmethod
+    def _new_dv_session(session, verbose=False):
+        response = session.get(DATAVIEWER_LMP_URL)
+        if verbose:
```

Comment:
our general pattern has been to do the verbose print before the request is made, so lets move this up

Reply:
Whoops, good catch